viewer/packages/360-images/src/collection/DefaultImage360Collection.ts

```diff
@@ -37,7 +37,7 @@ export class DefaultImage360Collection implements Image360Collection {
 
   private _needsRedraw: boolean = false;
 
-  private _defaultStyle: Image360AnnotationAppearance = {};
+  private readonly _defaultStyle: { appearance: Image360AnnotationAppearance } = { appearance: {} };
```

Comment:
Why this change? Is it planned to include more things?

Reply:
No, fixed :+1: